src/common/contracts/core/campaigns/client.ts

```
@@ -95,3 +103,10 @@
 
 export const can_process_refunds = (args: GetCampaignArgs) =>
   contractApi.view<typeof args, Campaign>("can_process_refunds", { args });
+
+export const storage_deposit = (depositAmountYocto: IndivisibleUnits) =>
+  contractApi.call<{}, IndivisibleUnits>("storage_deposit", {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)